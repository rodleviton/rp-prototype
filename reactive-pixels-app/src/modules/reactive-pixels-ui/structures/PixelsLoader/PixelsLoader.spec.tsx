import * as React from "react";
import { render } from "react-testing-library";
import WithRouter from "../../utils/test/WithRouter";
import WithTheme from "../../utils/test/WithTheme";
import { PixelsLoader } from "./";

describe("<PixelsLoader />", () => {
  it("should render <PixelsLoader /> component", () => {
    const { container } = render(
      <WithRouter>
        <WithTheme>
          <PixelsLoader />
        </WithTheme>
      </WithRouter>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
