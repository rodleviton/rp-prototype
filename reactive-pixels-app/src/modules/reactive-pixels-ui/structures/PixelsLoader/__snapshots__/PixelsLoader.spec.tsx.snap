// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PixelsLoader /> should render <PixelsLoader /> component 1`] = `
<div
  class="css-1x10l6g"
>
  <div
    class="css-1lz8w4n"
  >
    <div
      class="css-nnvb6c"
    >
      <svg
        height="38"
        viewBox="0 0 460 516"
        width="38"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="#9C9CA0"
        >
          <path
            class="css-1jnxzie"
            d="M333.372031,460 L447.102888,396.049894 C454.225767,391.973054 460,382.065204 460,373.932338 L460,249.593886 L356.224274,190 L355.657897,324.823726 C355.641368,328.729192 352.857585,333.480849 349.437,335.43865 L232,402.227074 L333.372031,460 Z"
          />
          <path
            class="css-1jnxzie"
            d="M210.509531,3.05293948 C217.583554,-1.02216524 233.04954,-1.01630663 240.11342,3.06249801 L447.204762,124.330903 C454.271377,128.411286 460,138.346818 460,146.527545 L460,246 L233.47922,115.241236 C230.076573,113.302921 224.591894,113.354919 221.221029,115.362032 L105,184.563481 L105,66.1376678 L210.509531,3.05293948 Z"
          />
          <path
            class="css-1jnxzie"
            d="M1,374.214594 C1,382.34674 5.78727134,392.236312 12.9351163,396.308633 L217.689432,512.962717 C224.8333,517.032772 236.399981,517.009813 243.528593,512.908989 L331,462.589963 L108.177655,334.741092 C104.750877,332.783693 101.972924,328.034824 101.972924,324.124429 L101.972924,68 L12.9872928,118.288501 C5.81460899,122.306552 0,132.156449 0,140.28838 L1,374.214594 Z"
          />
        </g>
      </svg>
    </div>
  </div>
  <div
    class="css-pvvspc"
  />
</div>
`;
